samples/HelloWorld/HelloWorld.al

```diff
@@ -11,6 +11,6 @@ codeunit 70051001 HelloWorld
     var
         HelloText : Codeunit GreetingsManagement;
     begin
-        Message(HelloText.GetRandomGreeting() + ', ' + Rec.Name);
+        Message('%1, %2',HelloText.GetRandomGreeting(),Rec.Name);
```

Comment:
@StanislawStempin — I agree with spaces after commas. Anything that improves readability is a good thing. @waldo1001 — I'd love to get rid of the %1, %2 style string substitution syntax and replace it with something better. Using PS syntax makes perfect sense.